src/main/java/practicalities/items/ItemMagnet.java

```diff
@@ -64,6 +67,8 @@ public void onUpdate(ItemStack item, World world, Entity entity, int i, boolean
 				this.distanceFromPlayer).iterator();
 		while (iterator.hasNext()) {
 			EntityItem itemToGet = (EntityItem) iterator.next();
+			if( PolarityNegatorManager.isEntityCloseToNegator(itemToGet) )
```

Comment:
That might work, though most magnets pull items, so they would be pulled out of the range.